modules/mono/editor/GodotTools/GodotTools/Utils/OS.cs

```diff
@@ -58,12 +69,28 @@ public static class Platforms
             [Names.HTML5] = Platforms.HTML5
         };
 
+        public static readonly Dictionary<string, string> DotNetOSPlatformMap = new Dictionary<string, string>
+        {
+            [Names.Windows] = DotNetOS.Win,
+            [Names.MacOS] = DotNetOS.OSX,
+            [Names.Linux] = DotNetOS.Linux,
+            // Does .NET 6 support BSD variants?
```

Comment:
> For the other BSD variants we might want to use unix instead of linux?

If that truly is needed, then it complicates things because Godot doesn't differentiate Linux and BSD variants when it comes to the export platform.

Reply:
I updated the comment into a TODO with this information included.